src/main/java/org/cascadebot/cascadebot/commands/music/SearchCommand.java

```diff
@@ -68,8 +74,12 @@ public void onCommand(Member sender, CommandContext context) {
                 messageBuilder.append('\n');
             }
 
-            EmbedBuilder embedBuilder = MessagingObjects.getMessageTypeEmbedBuilder(MessageType.INFO);
-            embedBuilder.setTitle(context.i18n("commands.search.multiple_results"));
+            EmbedBuilder embedBuilder = MessagingObjects.getMessageTypeEmbedBuilder(MessageType.INFO, context.getUser());
+            if (searchResults.size() > 1) {
+                embedBuilder.setTitle(context.i18n("commands.search.multiple_results"));
+            } else if (searchResults.size() == 1) {
+                embedBuilder.setTitle(context.i18n("commands.search.single_result"));
+            }
```

Comment:
This should really use the language stuff, and not have hardcoded values. same for 0 results

Reply:
Note for self: ICU Choose templating